woocommerce/woocommerce-android
# [Deposit Summary] Expanded UI version with animation

Closes: #10069

### Description

The PR adds UI that shows more data about deposits/balances. It also adds animation.

The PR missing:
* Dark theme support
* Tabs per currency
* Proper colors for the deposit statuses
* Deposit Scheduler is just hardcoded for now
* Learn more action

### Testing instructions

On the account with WCPay go to
* More -> Payments
* Notice the deposit info
* Click on it and notice the animation

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/505d7cc4-ec00-425b-bb4e-2cff3b7be3e5
